demo/sync/syncGlobalManager.js

```diff
@@ -20,6 +20,10 @@ function initSync() {
         return reject(error);
       }
       var cloudUrl = $fh.getCloudURL();
+
+      // decode the cloud app url if its encoded - required for keycloak
+      cloudUrl = decodeURIComponent(cloudUrl);
```

Comment:
Yeah.. We will use that url a lot in different places to get handle and do network call so better to provide some kind of general handler with explanation why it's needed rather than doing that for sync only.

Let's talk about it.